arpack-ng-config.cmake.in

```diff
@@ -20,6 +20,6 @@ set(includedir "@includedir@")
 # Create arpack targets.
 add_library(ARPACK::ARPACK INTERFACE IMPORTED)
 set_target_properties(ARPACK::ARPACK PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${includedir}/arpack")
-set_target_properties(ARPACK::ARPACK PROPERTIES INTERFACE_LINK_LIBRARIES      "arpack")
+set_target_properties(ARPACK::ARPACK PROPERTIES INTERFACE_LINK_LIBRARIES      "arpack@LIBSUFFIX@")
```

Comment:
Did you test that? Is the install dir usable through `find_package`? Why not "${libdir}/arpack..."?
Does this imply to change install `arpack` targets in CMakeLists.txt?